packages/dds/tree/src/simple-tree/tree.ts

```diff
@@ -81,24 +83,76 @@ export class TreeConfiguration<TSchema extends ImplicitFieldSchema = ImplicitFie
 
 /**
  * An editable view of a (version control style) branch of a shared tree.
+ *
+ * This view is always in one of three states:
```

Comment:
```suggestion
 * This view is always in one of two states:
```

I'm guessing you were originally going to split the failure states based on whether they can upgrade+recover in the documentation but decided against it?